mmv1/third_party/terraform/services/cloudrunv2/resource_cloud_run_v2_service_test.go

```diff
@@ -0,0 +1,184 @@
+package cloudrunv2_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccCloudRunV2Service_cloudrunv2ServiceFunctionExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"zip_path":      "./test-fixtures/function-source.zip",
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		//CheckDestroy:             testAccCheckCloudRunV2ServiceDestroyProducer(t),
```

Comment:
Was there trouble with the CheckDestroy? `testAccCheckCloudRunV2ServiceDestroyProducer()` should be autogenerated by the other tests, and would be used at the end of the test to verify that `terraform destroy` actually removed the service.